packages/client-search/src/types/HasPendingMappingsOptions.ts

```diff
@@ -0,0 +1,17 @@
+export type HasPendingMappingsOptions = {
+  /**
+   * If the clusters pending mapping state should be on the response.
+   *
+   * @alias getClusters
```

Comment:
if this is an alias, why do we support both?

Reply:
Actually, I was thinking about removing `alias` here, and marking `getClusters` as internal. WDYT?

Reply:
if they're just aliases, I'd even argue for not making getClusters an argument

Reply:
I will add getClusters  it as `@internal` so we have type here: https://github.com/algolia/algoliasearch-client-javascript/blob/858e43dfacef7d26d1a44b147741183b0e8677b3/packages/client-search/src/methods/client/hasPendingMappings.ts#L14